migrations/auth/1587982019_cleanup_columns.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE _auth_authenticator_oob DROP COLUMN activated;
+ALTER TABLE _auth_authenticator_oob DROP COLUMN activated_at;
+DROP TABLE _auth_authenticator_oob_code;
+ALTER TABLE _auth_identity_login_id DROP COLUMN password;
+ALTER TABLE _auth_identity_login_id DROP COLUMN realm;
+ALTER TABLE _auth_identity_oauth DROP COLUMN token_response;
```

Comment:
Since we updated some of the table schema here and the old code is not updated, so the api are broken. We need to unbreak it before we merge interaction back to master, I am ok for this. Just make sure we aware of it. cc @louischan-oursky 

And I found we drop `token_response` in oauth table here, seems we are not using it actually. Would like to confirm we don't need it?

Reply:
I think the token response is intended for supporting refreshing the User Info in the future if there is a refresh token in it.

Reply:
If we want to remove it, I prefer we remove all related code. I saw in Carmen's PR we simply do not save it in the store implementation to unbreak things. Therefore keeping it instead of removing it would minimize the size of code changes.

Reply:
Okay. I'll keep it and set it to an empty object for the new identity provider.